discord/bot.py

```diff
@@ -340,6 +342,26 @@ def command_group(self, name: str, description: str, guild_ids=None) -> SubComma
         self.add_application_command(group)
         return group
 
+    async def get_context(self, interaction: Interaction) -> InteractionContext:
```

Comment:
On second thought this probably would need to be something like `get_application_context`

Reply:
Yeah it should be renamed to that as well as a `cls` parameter for custom interaction context